usagov/vote-gov-drupal
# Vote-3056: usa-alert style update

## Jira ticket

[Vote-3056](https://cm-jira.usa.gov/browse/VOTE-3056)

## Description

This PR will address a style regression after the [most recent USWDS update](https://github.com/uswds/uswds/releases).  Also a code clean up to combine style sheets that were doing the same thing but also removed no longer needed spacing style due to the update for this component. 

**Before:** 
<img width="1329" alt="Screenshot 2024-10-31 at 10 46 51 AM" src="https://github.com/user-attachments/assets/865683b7-5acf-4ed3-ab95-d8149c7f73e0">

**After:**
<img width="1339" alt="Screenshot 2024-10-31 at 10 46 45 AM" src="https://github.com/user-attachments/assets/69009bb4-2677-41d1-ab8a-3aef1add707e">

 

## Deployment and testing

### Post-deploy steps

1. run `lando retune` and cd into the `votegov` theme and run `npm run build`

### QA/Testing instructions

1. Visit the site and ensure the site wide alert no longer has the style regression as found on the live site
2. Also create another site wide alert for emergency to verify no new style regressions
3. Fill out NVRF and verify that alerts in the form are still looking as expected

## Checklist for the Developer




## Checklist for the Peer Reviewers
